src/plugins/footnotes/footnotes-utils.ts

```diff
@@ -44,6 +46,7 @@ export const findTableInlineFootnoteIds = ($pos: ResolvedPos) => {
           tableElement,
           (node) => node.type === schema.nodes.inline_footnote
         )
+          .filter(({ node }) => !(isRejectedInsert(node) || isDeleted(node)))
```

Comment:
We want to keep the deleted `inline-footnote` nodes. It's expected for the label to stay there with a line through.